src/cli/MeasureManager.cpp

```diff
@@ -1001,6 +998,9 @@ void MeasureManagerServer::handle_request(const web::http::http_request& message
 }
 
 void MeasureManagerServer::do_tasks_forever() {
+  fmt::print("MeasureManager Ready");
+  fmt::print("Accepting requests on: {}\n", m_url);
+  std::cout << std::flush;
```

Comment:
Nitpick but having `fmt::print` followed by an old std::cout feels weird. 

```suggestion
  std::fflush(std::cout);
```

Or even `std::fflush(nullptr);` if you want to flush everything.

Reply:
I agree. It kind of smells to me also, but I don't think the fmt library has an API to flush. @kflemin was observing that the logs were coming in late, so I think we do actually need a flush for PAT to get the feedback it is looking for.

std::fflush(...) seems like an improvement to me over std::cout << std::flush. thanks!